website/conferences/views.py

```diff
@@ -241,7 +241,9 @@ def conference_view(**kwargs):
             continue
         meetings.append({
             'name': conf.name,
-            'active': conf.active,
+            'location': conf.location,
+            'end_date': conf.end_date.strftime("%d. %b %Y") if conf.end_date else None,
```

Comment:
The date format looks like this: `29. Aug 2015 `, but I don't think there should be a period after the day. Instead, I think it should look like this: `29 Aug 2015`